recipes/vcfsamplecompare/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "vcfsamplecompare" %}
+{% set realname = "vcfSampleCompare" %}
+{% set version = "v2.004" %}
+about:
+  home: https://github.com/hepcat72/{{ realname }}
+  license: GNU
+  license_file: LICENSE
+  summary: This script sorts and (optionally) filters the rows/variants of a VCF
+    file (containing data for 2 or more samples) based on the differences in the
+    variant data between samples or sample groups. Degree of "difference" is
+    determined by either the best possible degree of separation of sample groups
+    by genotype calls or the difference in average allelic frequency of each
+    sample or sample group (with a gap size threshold). The pair of samples or
+    sample groups used to represent the difference for a variant row is the one
+    leading to the greatest difference in consistent genotype or average allelic
+    frequencies (i.e. observation ratios, e.g. AO/DP) of the same variant state.
+    If sample groups are not specified, the pair of samples leading to the
+    greatest difference is greedily discovered and chosen to represent the
+    variant/row.
+
+build:
+  number: 0
+package:
+  name: {{ name }}
+  version: {{ version }}
+requirements:
+  build:
+    - perl
+    - make
+  run:
+    - perl
+
+source:
+  url: https://github.com/hepcat72/{{ realname }}/archive/{{ version }}.tar.gz
+  sha256: 646515313907441ea8540c359baf552682fbab27396abadac3ff8b6c4402d71c
+
+test:
+  commands:
+    - {{ realname }}.pl --usage
+    - {{ realname }}.pl --version
+    - {{ realname }}.pl --help
```

Comment:
I think that just `--version` should be enough.

Reply:
Done.